locking/locks.go

```diff
@@ -154,6 +157,17 @@ func (c *Client) UnlockFileById(id string, force bool) error {
 	if err := c.cache.RemoveById(id); err != nil {
 		return fmt.Errorf("Error caching unlock information: %v", err)
 	}
+	path := unlockRes.Lock.Path
+	gitRoot, err := git.RootDir()
+	if err != nil {
+		return err
+	}
+	absPath := filepath.Join(gitRoot, path)
+
+	// Make non-writeable if required
+	if c.SetLockableFilesReadOnly && c.IsFileLockable(path) {
```

Comment:
Good catch.

Can you add a test in the `test/test-unlock.sh` file to prevent this functionality from regressing in the future and add a note to the manpage to document this behavior?